Authentication/readme.md

```diff
@@ -0,0 +1,110 @@
+# Juntos Auth Service
+
+## This Authentication module will provide basic auth functionality as follows:-
```

Comment:
Please use appropriate structure.
This is the module description. I don't think this should be a heading.